Essentials/src/com/earth2me/essentials/commands/Commandtime.java

```diff
@@ -166,25 +130,29 @@ private String normalizeWorldName(World world) {
 
     @Override
     protected List<String> getTabCompleteOptions(Server server, CommandSource sender, String commandLabel, String[] args) {
-        final User user = ess.getUser(sender.getPlayer());
-
+        boolean isConsole = sender.isPlayer();
```

Comment:
This logic doesn't seem right to me

Reply:
What logic?

Reply:
if the sender is a player, they are not console.

Reply:
oh lmao, fixed i fixed that up